homeassistant/core.py

```diff
@@ -595,7 +595,7 @@ def async_add_hass_job(
                 hassjob.target = cast(
                     Callable[..., Coroutine[Any, Any, _R]], hassjob.target
                 )
-            task = self.loop.create_task(hassjob.target(*args), name=hassjob.name)
```

Comment:
I disagree with this change. `asyncio.create_task` results in the same function call on `self.loop`, just with more overhead. Given that this the absolute core of HA (everything runs through here), I don't see the need of adding this overhead to adhere to best recommendations.

But I will leave it up to @bdraco to make the final decision, as he is the king of performance.